File: base/Sources/export_mesh.ts

function export_mesh_run(path: string, paint_objects: mesh_object_t[] = null, apply_disp: bool = false) {
	if (paint_objects == null) {
		paint_objects = project_paint_objects;
	}
	if (context_raw.export_mesh_format == mesh_format_t.OBJ) {
		export_obj_run(path, paint_objects, apply_disp);
	}
	else {
		export_arm_run_mesh(path, paint_objects);
	}
}

File: paint/sources/export_mesh.c

#include "global.h"

void export_mesh_run(char *path, mesh_object_t_array_t *paint_objects, bool apply_disp, bool merge_vertices) {
	if (paint_objects == NULL) {
		paint_objects = project_paint_objects;
	}
	if (g_context->export_mesh_format == MESH_FORMAT_OBJ) {
		merge_vertices ? export_obj_run(path, paint_objects, apply_disp) : export_obj_run_fast(path, paint_objects);
	}
	else {
		export_arm_run_mesh(path, paint_objects);
	}
}
